### ISSUE TITLE
FR: Command Line Option to show versions

### ISSUE THREAD

**Issue by Steve-Mcl** [node-red/node-red#4569]
TLDR; 
* Add support for `node-red --version` that returns the version number e.g. `v3.1.5`
* Add support for `node-red --info` that returns version numbers of node, node-red, npm and OS info

A reasonable request to show node-red version via command line was requested here: https://discourse.nodered.org/t/command-line-to-show-all-versions-running/85637

To add to that, the request also asks that additional (relevant) info be returned too e.g. 

```
uname -r
node -v
npm -v
node-red --version
```

As it is quite specific / different from a normal CLI where `--version` would be the norm, perhaps we could support this via something like `--info`?

NOTE: windows batch file should use `ver` instead of `uname -r` (it returns, for example "Microsoft Windows [Version 10.0.nnnnn.nnnn]")

NOTE: Trying to keep this simple as possible but useful for support folk who need to know specific info when triaging issues.

**Comment by knolleary** [comment]
Closed via #4707
